torchvision/datasets/widerface.py

```diff
@@ -181,3 +170,15 @@ def download(self) -> None:
         download_and_extract_archive(url=self.ANNOTATIONS_FILE[0],
                                      download_root=self.root,
                                      md5=self.ANNOTATIONS_FILE[1])
+
+
+def check_integrity(self) -> bool:
```

Comment:
That's not great, but for now `DatasetTestCase` cannot patch the `_check_integrity` method, so I had to move it as a function.

What would you recommend here @pmeier? I would tend towards updating `DatasetTestCase._patch_checks()` instead, as it's a bit unfortunate to have to change some code here just so that it's compatible with the test infrastructure

Reply:
Why do you need to patch the `_verify_integrity` method? Shouldn't this simply return `True` if the fake data is correctly set up?